apps/laboratory/tests/shared/pages/WalletPage.ts

```diff
@@ -33,7 +35,15 @@ export class WalletPage {
     if (isVercelPreview) {
       await this.vercelPreview.evaluate((iframe: HTMLIFrameElement) => iframe.remove())
     }
-    await this.gotoHome.click()
+    /*
+     * If connecting to a single account manually navigate.
+     * Otherwise click the home button.
+     */
+    if (this.connectToSingleAccount) {
+      await this.page.goto(`${this.baseURL}/walletconnect?addressesToApprove=1`)
```

Comment:
Me and Gancho added `addressesToApprove` query so we can connect to single account